SwiftUIDZ/ContentView.swift

```diff
@@ -7,16 +7,189 @@
+    @State private var isActionSheetShown = false
+    @State private var isLuxaryInteriorCar = false
+    @State private var isEditingSlider = false
+    @State private var isSheetShowh = false
+    @State private var carCosts: [Float] = Constants.carCosts
+    
+    private var cars = Constants.cars
+    private var carNames = Constants.carsFullNames
+    private var carSpecifications = Constants.carSpecifications
+    
+    private var carPicker: some View {
+        Picker(selection: $pickerIndex) {
+            ForEach(0 ..< cars.count) {
+                Text(cars[$0]).tag($0)
+            }
+        } label: {
+            Text(Constants.emptyText)
+        }
+        .pickerStyle(.segmented)
+    }
+    
+    private var carImage: some View {
+        Image(cars[pickerIndex])
+            .resizable()
+            .cornerRadius(Constants.defaultCornerRadius)
+            .aspectRatio(Constants.aspectRatio, contentMode: .fit)
+    }
+    
+    private var carNameText: some View {
+        Text(carNames[pickerIndex])
+            .font(Font.title)
+    }
+    
+    private var buyButton: some View {
+        Button(Constants.buyButtonName) {
+            isAlertShown = true
+        }
+        .alert(isPresented: $isAlertShown, content: {
+            Alert(title: Text(Constants.alertTitle),
+                  dismissButton: .default(Text(Constants.okText)))
+        })
+        .frame(width: Constants.defaultWidth, height: Constants.defaultHeight)
+        .foregroundColor(.white)
+        .background(Color.gray)
+        .cornerRadius(Constants.defaultCornerRadius)
+    }
+    
+    private var feedbackButton: some View {
+        Button(Constants.feedbackButtonName) {
+            isActionSheetShown = true
+        }
+        .actionSheet(isPresented: $isActionSheetShown, content: {
+            ActionSheet(title: Text(Constants.actionSheetTitle))
+        })
+        .frame(width: Constants.defaultWidth, height: Constants.defaultHeight)
+        .foregroundColor(.white)
+        .background(Color.gray)
+        .cornerRadius(Constants.defaultCornerRadius)
+    }
+    
+    private var shareButton: some View {
+        Button(Constants.shareButtonName) {
+            isSheetShowh = true
+        }
+        .sheet(isPresented: $isSheetShowh, content: {
+            let shareText = """
+                \(carNames[pickerIndex])
+                \(carSpecifications[pickerIndex])
+                \(carCosts[pickerIndex]) \(Constants.rubleText)
+                """
+            ActivityView(activityItems: [shareText])
+        })
+        .frame(width: Constants.defaultWidth, height: Constants.defaultHeight)
+        .foregroundColor(.white)
+        .background(Color.gray)
+        .cornerRadius(Constants.defaultCornerRadius)
+    }
+    
+    private var luxaryInteriorCarToggle: some View {
+        Toggle(isOn: Binding(get: {
+            isLuxaryInteriorCar
+        }, set: { newValue in
+            if newValue {
```

Comment:
```suggestion
            if newValue {
```
это бизнес логика она должна быть во viewModel